sleap_io/model/skeleton.py

```diff
@@ -22,6 +22,14 @@ class Node:
 
     name: str
 
+    @staticmethod
+    def from_names(name_list: List[str]) -> List["Node"]:
+        """Convert list of node names to list of nodes objects."""
+        nodes = list()
+        for name in name_list:
+            nodes.append(Node(name))
+        return nodes
+
```

Comment:
Remove this - it's back from the dead (no changes should be made to `models` directory for this PR)